docs/manager/README.md

```diff
@@ -1122,7 +1114,7 @@ All errors will be displayed in standard output and logged into `/var/log/clwpos
     rpm -q ea-php74-php-redis
     yum install ea-php74-php-redis
     ```
-8. Check Redis `ini` file is present by running the following command:
+2. Check Redis `ini` file is present by running the following command (example for cPanel):
```

Comment:
The whole section is called "Ensuring the EA-PHP Redis extension is configured correctly", thus already telling that it is cPanel-specific (ea-php packages are for cPanel only).
So, this note lloks redundant.

Reply:
removed